local/build-binary

```diff
@@ -152,8 +151,8 @@ pushd ${WORKDIR}
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
         -DWITH_TOKUDB=${WITH_TOKUDB} \
         -DWITH_ROCKSDB=${WITH_ROCKSDB} \
-        -DWITH_SSL=${WITH_SSL} \
         -DWITH_MECAB=${WITH_MECAB} \
+        -DWITH_SYSTEM=libs \
```

Comment:
It's actually -DWITH_SYSTEM_LIBS=ON https://dev.mysql.com/doc/refman/8.0/en/source-configuration-options.html#option_cmake_with_system_libs